tests/science/global_param.classic.5.0_snotel.txt

```diff
@@ -0,0 +1,94 @@
+NLAYER                3
+NODES                 10
+MODEL_STEPS_PER_DAY   24
+SNOW_STEPS_PER_DAY    24
+RUNOFF_STEPS_PER_DAY  24
+STARTYEAR             2007
+STARTMONTH            11
+STARTDAY              01
+STARTSEC              00
+ENDYEAR               2008
+ENDMONTH              07
+ENDDAY                31
+
+FULL_ENERGY           TRUE
+FROZEN_SOIL           FALSE
+
+QUICK_FLUX            FALSE
+
+FORCING1              /Users/diana/Dropbox/UW/VIC_test_data/datasets/snotel/forcings_vp/snotel_VIC.4.1.2_forcings_
+FORCE_FORMAT          ASCII
+FORCE_TYPE            SWDOWN
+FORCE_TYPE            LWDOWN
+FORCE_TYPE            AIR_TEMP
+FORCE_TYPE            PRESSURE
+FORCE_TYPE            PREC
+FORCE_TYPE            VP
+FORCE_TYPE            WIND
+
+# FORCE_TYPE            SKIP
+
+FORCE_STEPS_PER_DAY   24
+FORCEYEAR             2007
+FORCEMONTH            11
+FORCEDAY              01
+GRID_DECIMAL          4
+WIND_H                17.5
+
+SOIL                  /Users/diana/Dropbox/UW/VIC_test_data/datasets/snotel/parameters/Snotel_VIC.4.1.2_soil.txt
+BASEFLOW              ARNO
+
+JULY_TAVG_SUPPLIED    FALSE
+ORGANIC_FRACT         FALSE
+
+VEGLIB                /Users/diana/Dropbox/UW/VIC_test_data/datasets/snotel/parameters/Snotel_VIC.4.1.2_veglib.txt
+VEGPARAM              /Users/diana/Dropbox/UW/VIC_test_data/datasets/snotel/parameters/Snotel_VIC.4.1.2_vegparam.txt
+ROOT_ZONES            3
+
+VEGPARAM_LAI          TRUE
+LAI_SRC               FROM_VEGLIB
```

Comment:
@jhamman, I've changed `LAI_SRC` to `LAI_FROM_VEGPARAM` to be consistent.  

@bartnijssen, indeed - this was an initial debugging commit to ensure that we had the same options set for both versions. I've removed the hard-coded paths.